libcodechecker/server/api/report_server.py

```diff
@@ -460,6 +506,7 @@ def __init__(self,
         self.__suppress_handler = suppress_handler
         self.__package_version = package_version
         self.__Session = Session
+        self.Session = Session
```

Comment:
Why do we need this? We have a private variable above (`self.__Session`) which we have used in other parts of the code.